sdk/identity/azure-identity/src/main/java/com/azure/identity/IdentityClient.java

```diff
@@ -285,4 +416,16 @@ private static void sleep(int millis) {
             throw new RuntimeException(ex);
         }
     }
+
+    private static Proxy proxyOptionsToJavaNetProxy(ProxyOptions options) {
+        switch (options.type()) {
+            case HTTP:
+                return new Proxy(Type.HTTP, options.address());
+            case SOCKS4:
+            case SOCKS5:
+                return new Proxy(Type.SOCKS, options.address());
+            default:
+                return new Proxy(Type.HTTP, options.address());
```

Comment:
Move the HTTP case above the default one and fall through, rather than have the same line of code in two places (this is a good place for bugs to creep in).